src/view/trip-point-form/trip-pictures.js

```diff
@@ -0,0 +1,33 @@
+import AbstractView from '../abstract';
+
+const createPictures = (task) => {
+  const { destination } = task;
+  let items = '';
+  for (let actionIndex = 0; actionIndex < destination.pictures.length; actionIndex++) {
+    items += `<img class="event__photo" src="${destination.pictures[actionIndex]}" alt="Event photo">`;
+  }
+  return items;
+};
+
+const createPicturesContainer = (task) => {
```

Comment:
```suggestion
const createPicturesContainerTemplate = (task) => {
```

по функции будет понятннее что она делает